sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -144,6 +155,17 @@ def load(
     :paramtype on_refresh_error: Optional[Callable[[Exception], None]]
     :keyword on_refresh_error: Optional callback to be invoked when an error occurs while refreshing settings. If not
     specified, errors will be raised.
+    :paramtype feature_flag_enabled: bool
+    :keyword feature_flag_enabled: Optional flag to enable or disable the loading of feature flags. Default is False.
+    :paramtype feature_flag_selectors: List[SettingSelector]
+    :keyword feature_flag_selectors: Optional list of selectors to filter feature flags. By default will load all
+    feature flags without a label.
+    :paramtype feature_flag_refresh_enabled: bool
+    :keyword feature_flag_refresh_enabled: Optional flag to enable or disable the refresh of feature flags. Default is
+     False.
+    :paramtype feature_flag_trim_prefixes: List[str]
+    :keyword feature_flag_trim_prefixes: Optional list of prefixes to trim from feature flag keys. By default will trim
+     the FEATURE_FLAG_PREFIX.
```

Comment:
With how the [trimming logic is currently](https://github.com/Azure/azure-sdk-for-python/pull/33411/files/83d98f860cf3600b69d0ebb7ec3b991428e097f8#diff-6d4c012a959a18eeaa7686743d07b6249f2b3be4c8ad5b2b45594756c2f2112dR646 ), I think this description isn't quite accurate. 

It always trims the "FEATURE_FLAG_PREFIX" and in addition, will trim the first feature_flag_trim_prefixes it runs into. I think either approach works. I slightly prefer updating this description to be something like:
```
After the FEATURE_FLAG_PREFIX is trimmed, the first match in feature_flag_trim_prefixes will be trimmed, if there is one.
```